src/util/constants.ts

```diff
@@ -20,127 +20,157 @@ export const baseEndpoints = {
   CDN_URL: IMAGE_BASE_URL,
 };
 
-const GUILDS_BASE = (id: string) => `${baseEndpoints.BASE_URL}/guilds/${id}`;
+const GUILDS_BASE = (guildID: string) =>
+  `${baseEndpoints.BASE_URL}/guilds/${guildID}`;
+const CHANNEL_BASE = (channelID: string) =>
+  `${baseEndpoints.BASE_URL}/channels/${channelID}`;
 
 export const endpoints = {
+  GUILDS_BASE: GUILDS_BASE,
+  CHANNEL_BASE: CHANNEL_BASE,
```

Comment:
```suggestion
  GUILDS_BASE,
  CHANNEL_BASE,
```
Nitpick, but you can't just do this.